content/p2p_sync_intro.tex

```diff
@@ -44,6 +45,7 @@
     \TableCaptionRef{Point-to-Point Synchronization Types and Names}
     \label{p2psynctypes}
   \end{center}
+\end{DeprecateBlock}
```

Comment:
There should be a sentence either in the backmatter deprecation rationale or somewhere "nearby" this table that talks about why this _entire_ table is marked deprecated. Otherwise, why have this table at all if there aren't any outstanding references to it now.

Reply:
There are still references to it:

https://github.com/naveen-rn/specification/blob/4cc60ba76369f910638076ba1195ae1bf5b4f8ee/content/shmem_wait_until.tex#L19-L28

I guess one could ask, do we need to deprecate the table itself if all the interfaces the reference it are deprecated?

Reply:
@npark, @BryantLam Yes - I would think deprecating the table is appropriate to avoid user's being confused about which table to look at when using the non-deprecated p2p sync routines.

Would it help to add a row to the deprecation table regarding the p2p type-table deprecation?  Perhaps similar to:
```
    \minitab{Table~\ref{p2psynctypes}: point-to-point synchronization types}
        & 1.5 & Current & Table~\ref{stdamotypes}: standard AMO types \\ \hline
```

I also drafted a rationale section for that:
```
\subsection{Table~\ref{p2psynctypes}: point-to-point synchronization types}
The point-to-point synchronization routines are only compatible with \acp{AMO}
(as of \openshmem 1.5), so all point-to-point synchronization types must be
compatible with the \ac{AMO} types.  Accordingly, all non-deprecated
point-to-point synchronization interfaces are defined by referring to the standard \ac{AMO} type
table.
```

Do you want to add it, or is it ok without?

Reply:
If there are still outstanding references to it, we should leave in the p2psynctypes table.

I'm okay with adding this entry into the deprecation table + rationale. I like that it makes it clear why the table is deprecated. The last sentence of the rationale reads a bit weird, so maybe it could be rephrased. Otherwise it looks good!

Reply:
It looks like @Bryant kind of approved this PR :) I'm fine with the deparate table + rationale approach.